resources/assets/coffee/react/_components/beatmap-icon.coffee

```diff
@@ -37,7 +37,9 @@ el = React.createElement
 
     showTitle = @props.showTitle && !@props.overrideVersion?
 
+    mode = if beatmap.convert then 'osu' else beatmap.mode
```

Comment:
um, well, whatever ┐(-｡ｰ;)┌

<sub>Though I'd personally be sad if I have beatmapset and see converted beatmap listed before my own beatmaps. Not that I map anything. Also it's weird seeing last beatmap selected when switching mode. Might as well sort it descending.</sub>